egs2/TEMPLATE/vc1/vc.sh

```diff
@@ -0,0 +1,830 @@
+
+            key_file="${_valid_dir}/${_scp}"
+            split_scps=""
+            for n in $(seq "${_nj}"); do
+                split_scps+=" ${_logdir}/valid.${n}.scp"
+            done
+            # shellcheck disable=SC2086
+            utils/split_scp.pl "${key_file}" ${split_scps}
+
+            # 2. Generate run.sh
+            log "Generate '${vc_stats_dir}/run.sh'. You can resume the process from stage 5 using this script"
+            mkdir -p "${vc_stats_dir}"; echo "${run_args} --stage 5 \"\$@\"; exit \$?" > "${vc_stats_dir}/run.sh"; chmod +x "${vc_stats_dir}/run.sh"
+
+            # 3. Submit jobs
+            log "VC collect_stats started... log: '${_logdir}/stats.*.log'"
+            # shellcheck disable=SC2086
+            ${train_cmd} JOB=1:"${_nj}" "${_logdir}"/stats.JOB.log \
+                ${python} -m "espnet2.bin.vc_train" \
+                    --collect_stats true \
+                    --write_collected_feats "${write_collected_feats}" \
+                    --use_preprocessor true \
+                    --non_linguistic_symbols "${nlsyms_txt}" \
+                    --normalize none \
+                    --pitch_normalize none \
+                    --energy_normalize none \
+                    --train_data_path_and_name_and_type "${_train_dir}/text,text,text" \
+                    --train_data_path_and_name_and_type "${_train_dir}/${_scp},speech,${_type}" \
+                    --valid_data_path_and_name_and_type "${_valid_dir}/text,text,text" \
+                    --valid_data_path_and_name_and_type "${_valid_dir}/${_scp},speech,${_type}" \
+                    --train_shape_file "${_logdir}/train.JOB.scp" \
+                    --valid_shape_file "${_logdir}/valid.JOB.scp" \
+                    --output_dir "${_logdir}/stats.JOB" \
+                    ${_opts} ${train_args} || { cat "${_logdir}"/stats.1.log; exit 1; }
+
+            # 4. Aggregate shape files
+            _opts=
+            for i in $(seq "${_nj}"); do
+                _opts+="--input_dir ${_logdir}/stats.${i} "
+            done
+            ${python} -m espnet2.bin.aggregate_stats_dirs ${_opts} --output_dir "${vc_stats_dir}"
+
+            mv "${vc_stats_dir}/train" "${vc_stats_dir}/${_set_id}_train"
+            mv "${vc_stats_dir}/valid" "${vc_stats_dir}/${_set_id}_valid"
+        done
+    fi
+
+    if [ ${stage} -le 6 ] && [ ${stop_stage} -ge 6 ]; then
+        log "Stage 6: Combine Source and Target Datasets"
+
+        ${python} -m espnet2.vc.combine_src_trg \
+              --srcspk "${srcspk}" --trgspk "${trgspk}" \
+              --src_dir "${data_feats}/${src_train_set}" \
+              --trg_dir "${data_feats}/${trg_train_set}"
+
+        ${python} -m espnet2.vc.combine_src_trg \
+              --srcspk "${srcspk}" --trgspk "${trgspk}" \
+              --src_dir "${data_feats}/${src_valid_set}" \
+              --trg_dir "${data_feats}/${trg_valid_set}"
+
+        for i in "${!src_test_sets[@]}"; do
```

Comment:
I think `src_test_sets` is not array, which is just string separated with space.
Does it work?